docs/develop/json-manifest-overview.md

```diff
@@ -0,0 +1,871 @@
+
+|JSON property|Purpose|XML element(s)|Comments|
+|:-----|:-----|:-----|:-----|
+|"$schema"| Identifies the manifest schema. | attributes of **OfficeApp** and **VersionOverrides** | |
+|"id"| GUID of the add-in. | **Id**| |
+|"version"| Version of the add-in. | **Version** | |
+|"manifestVersion"| Version of the manifest schema. |  attributes of **OfficeApp** | |
+|"name"| Public name of the add-in. | **DisplayName** | |
+|"description"| Public description of the add-in.  | **Description** | |
+|"accentColor"||| This property has no equivalent in the current XML manifest and is not used in the preview of the JSON manifest. But it must be present. |
+|"developer"| Identifies the developer of the add-in. | **ProviderName** | |
+|"localizationInfo"| Configures the default locale and other supported locales. | **DefaultLocale** and **Override** | |
+|"webApplicationInfo"| Identifies the add-in's web app as it is known in Azure Active Directory. | **WebApplicationInfo** | In the current XML manifest, the **WebApplicationInfo** element is inside **VersionOverrides**, not the base manifest. |
+|"authorization"| Identifies any Microsoft Graph permissions that the add-in needs. | **WebApplicationInfo** | See comment in preceding row. |
+
+The **Hosts**, **Requirements**, and **ExtendedOverrides** elements are part of the base manifest in the current XML manifest. But concepts and purposes associated with these elements are configured inside the "extension" property of the preview JSON manifest. 
+
+#### "extension" property
+
+The "extension" property in the preview JSON manifest primarily represents characteristics of the add-in that would not be relevant to other kinds of Microsoft 365 extensions. For example, the Office applications that the add-in extends (such as, Excel, PowerPoint, Word, and Outlook) are specified inside the "extension" property as are customizations of the Office application ribbon. The property's purposes closely match those of the **VersionOverrides** element in the current XML manifest.
+
+> [!NOTE]
+> The **VersionOverrides** section of the manifest has a "double jump" system for many string resources. Strings, including URLs, are specified and assigned an ID in the **Resources** child of **VersionOverrides**. Elements that require a string have a `resid` attribute that matches the ID of a string in the **Resources** element. The "extension" property of the preview JSON manifest simplifies things by defining strings directly as property values. There is nothing equivalent to the **Resources** element.
```

Comment:
Is this note about the current manifest?

Reply:
```suggestion
> The **VersionOverrides** section of the current XML manifest has a "double jump" system for many string resources. Strings, including URLs, are specified and assigned an ID in the **Resources** child of **VersionOverrides**. Elements that require a string have a `resid` attribute that matches the ID of a string in the **Resources** element. The "extension" property of the preview JSON manifest simplifies things by defining strings directly as property values. There is nothing in the JSON manifest that is equivalent to the **Resources** element.
```